File: solutions/1603/1603-kir3i.java
class ParkingSystem {
    public int[] cap = new int[4];
    public int[] now = {0, 0, 0, 0};
    public ParkingSystem(int big, int medium, int small) {
        cap[1] = big;
        cap[2] = medium;
        cap[3] = small;
    }
    
    public boolean addCar(int carType) {
        if(now[carType] < cap[carType]) {
            now[carType]++;
            return true;
        }
        return false;
    }
}

File: solutions/1603/1603-kir3i.py
class ParkingSystem:
    cap = {}
    now = {}
    def __init__(self, big, medium, small):
        self.cap[3] = small
        self.cap[2] = medium
        self.cap[1] = big
        self.now[1] = 0
        self.now[2] = 0
        self.now[3] = 0
    def addCar(self, carType):
        if self.now[carType] < self.cap[carType]:
            self.now[carType] += 1
            return True
        else:
            return False
